controllers/category_controller.py

```diff
@@ -41,25 +42,34 @@ def add_category() -> Category:
     session.commit()
     return new_category
 
+class CategoryMenuEnums(IntEnum):
+    PCADD = 1
+    PCREMOVE = 2
+    PCRENAME = 3
+    CADD = 4
+    BUDGETTOCAT = 5
+    CRENAME = 6
+    BACKTOBUDGET = 7

```

Comment:
Na pierwszy rzut oka ciężko przeczytać te nazwy. Może lepiej PC_ADD, PC_REMOVE, itd?
BUDGETTOCAT też mało jasne, ale chyba przez to, że nasz budżet jest używany dwa razy. Jako rodzić parent category i jako pole w category :(